### DISCUSSION TITLE
第2节课作业(18班)

### DISCUSSION THREAD

**Discussion by mattheliu** [InternLM/Tutorial#98] [作业]
请提交作业的链接
Please submit a link to the assignment.

**Comment by yestolife** [comment]
**基础作业**：

- 使用 InternLM-Chat-7B 模型生成 300 字的小故事（需截图）。
  
<img width="677" alt="1" src="https://github.com/InternLM/tutorial/assets/34405348/6686fd8c-1e8f-4a39-8115-833735fe6b11">

  
- 熟悉 hugging face 下载功能，使用 huggingface_hub python 包，下载 InternLM-20B 的 config.json 文件到本地（需截图下载过程）。
  
<img width="737" alt="2" src="https://github.com/InternLM/tutorial/assets/34405348/c374cd68-0227-4acf-98a8-c1449aa9e4f4">
  

**进阶作业（可选做）**

- 完成浦语·灵笔的图文理解及创作部署（需截图）

<img width="947" alt="4" src="https://github.com/InternLM/tutorial/assets/34405348/fdd61f47-0c90-4e61-b504-4137848cdbce">
<img width="953" alt="5" src="https://github.com/InternLM/tutorial/assets/34405348/b536cf2f-8477-4beb-ad84-e6513a2dc17b">

- 完成 Lagent 工具调用 Demo 创作部署（需截图）

<img width="957" alt="3" src="https://github.com/InternLM/tutorial/assets/34405348/69c87bd5-41b4-4e3d-8630-2e465a1c7763">

**Comment by tackhwa** [comment]
基础作业：

- 使用 InternLM-Chat-7B 模型生成 300 字的小故事（需截图）。
![image](https://github.com/InternLM/tutorial/assets/55059307/36e9b27f-61fc-473d-b90e-da533d56a0e3)

- 熟悉 hugging face 下载功能，使用 huggingface_hub python 包，下载 InternLM-20B 的 config.json 文件到本地（需截图下载过程）。
![image](https://github.com/InternLM/tutorial/assets/55059307/3934a990-a2af-47c2-afe6-9a2cc9cebbac)
![image](https://github.com/InternLM/tutorial/assets/55059307/d1a9214d-8889-4249-8997-adfb5bdc4aa7)
![image](https://github.com/InternLM/tutorial/assets/55059307/88eabf04-167d-448f-b406-0a5e1ef477c5)

进阶作业（可选做）

- 完成浦语·灵笔的图文理解及创作部署（需截图）
-可能是由于小模型的限制（7b）,提供的答案还是有错误
![image](https://github.com/InternLM/tutorial/assets/55059307/2040e90c-f2d3-457e-ac51-c018387f38e7)
![image](https://github.com/InternLM/tutorial/assets/55059307/fafa390c-979b-4d29-9868-62770b2d8524)
![image](https://github.com/InternLM/tutorial/assets/55059307/1e536551-f97a-4ba6-b78a-59cd55941670)
![image](https://github.com/InternLM/tutorial/assets/55059307/bceddd8c-3345-47cb-b3ce-67eeaf269757)
![image](https://github.com/InternLM/tutorial/assets/55059307/aea16256-a364-459c-81eb-977f4c32833b)
![image](https://github.com/InternLM/tutorial/assets/55059307/dde74f19-b713-44aa-a19a-6b54be95b41f)

- 完成 Lagent 工具调用 Demo 创作部署（需截图）
![image](https://github.com/InternLM/tutorial/assets/55059307/b89b9fdf-829c-4a32-b13e-4d57369954c2)

**Comment by Heitao5200** [comment]
https://github.com/Heitao5200/my_blog/blob/master/docs/组队学习/大模型实战/TASK2.md

**Comment by mattheliu** [comment]
https://github.com/mattheliu/gitblog/issues/2

**Comment by caohuikang** [comment]
使用 InternLM-Chat-7B 模型生成 300 字的小故事:
![300字作文](https://github.com/InternLM/tutorial/assets/22697135/d9926f7b-b19b-40bb-ab96-01c7fe220d4c)

**Reply by caohuikang** [reply]
熟悉 hugging face 下载功能
![huggingface_code](https://github.com/InternLM/tutorial/assets/22697135/f045a11f-6674-41ea-b872-cea46ac3175b)
![huggingface_dw](https://github.com/InternLM/tutorial/assets/22697135/8da5f4ee-1d06-4cb0-a392-8e4e2213efa9)

**Comment by DrZhang1208** [comment]
[书生·浦语第二次作业](https://blog.csdn.net/qq_42866033/article/details/135442273?spm=1001.2014.3001.5502)

**Comment by ZMcc18** [comment]
https://www.yuque.com/zixul/avxnzn/tpehhouh2qt8mwgm?singleDoc# 《lesson 2 书生·浦语大模型 Demo作业》

**Comment by xwhclaire** [comment]
[Assignment 2 轻松玩转书生·浦语大模型趣味Demo](https://github.com/xwhclaire/StudyPackages/blob/main/LLM%20training/%23%20Assignment%202%20%E8%BD%BB%E6%9D%BE%E7%8E%A9%E8%BD%AC%E4%B9%A6%E7%94%9F%C2%B7%E6%B5%A6%E8%AF%AD%E5%A4%A7%E6%A8%A1%E5%9E%8B%E8%B6%A3%E5%91%B3Demo.md)

**Comment by ggfengxuan** [comment]
[Ⅱ.书生·浦语大模型趣味Demo作业](https://qa5023amair.feishu.cn/wiki/CMxOwoiRYijuqAkixiTcoxMxn3M?from=from_copylink)

**Comment by D-JI-Feng** [comment]
作业放在飞书云文档https://k0ilmnk4dz3.feishu.cn/wiki/QXnswm5GGiMFQCk5uLTcN504nDg?from=from_copylink

**Comment by ZiHAO-LI-cmd** [comment]
[书生·浦语大模型实战营 Lesson 2](http://t.csdnimg.cn/ZhFzr)

**Comment by scorpionfree98** [comment]
http://t.csdnimg.cn/AGNTg

**Comment by zxp22** [comment]
https://zhuanlan.zhihu.com/p/677151444

**Comment by 2001wjh** [comment]
https://www.yuque.com/wjh2001/rdvfen/ygqlm1vf5cuuwaav?singleDoc# 《轻松玩转书生·浦语大模型趣味Demo》
